clang/include/clang/CIR/Dialect/Builder/CIRBaseBuilder.h

```diff
@@ -404,6 +404,16 @@ class CIRBaseBuilderTy : public mlir::OpBuilder {
                                  mlir::ValueRange value = {}) {
     return create<mlir::cir::YieldOp>(loc, value);
   }
+
+  template <typename... Args>
```

Comment:
Whenever we fail to call `createCallOp` the error message is going to be even more annoying (i.e. hard to provide useful info) than when calls to `create<mlir::cir::CallOp>` are wrong. Please don't use templates here. Feel free to add multiple version if you need (with or without default args), and make them call into each other. The important part is that only one of them does the final `create<mlir::cir::CallOp>` call.